public/app/routes/routes.tsx

```diff
@@ -374,14 +374,14 @@ export function getAppRoutes(): RouteDescriptor[] {
       path: '/alerting/new',
       pageClass: 'page-alerting',
       component: SafeDynamicImport(
-        () => import(/* webpackChunkName: "NgAlertingPage"*/ 'app/features/alerting/NextGenAlertingPage')
+        () => import(/* webpackChunkName: "NgAlertingPage"*/ 'app/features/alerting/unified/AlertEditor')
       ),
     },
     {
       path: '/alerting/:id/edit',
```

Comment:
oh boy, this will need thinking about.. cloud alerts don't have ids ;/

Reply:
Yes this may be a good topic for discussing with the backend/API folks to see if we can create some sort of uuid for alert rules. Perhaps a hash of the namespace + group + alert name ?